tests/flow/test_results.py

```diff
@@ -0,0 +1,112 @@
+import os
```

Comment:
I'm not expecting this to work, or any other flow test for that matter until we update both our client(s) and our unit-tests